internal/pod_models.go

```diff
@@ -195,7 +184,7 @@ func GetPodSpec(cluster *fdbtypes.FoundationDBCluster, processClass fdbtypes.Pro
 		versionString = cluster.Spec.Version
 	}
 
-	image, err := GetImage(cluster.Spec.MainContainer.ImageName, mainContainer.Image, "foundationdb/foundationdb", versionString, processSettings.GetAllowTagOverride())
+	image, err := GetImage(mainContainer.Image, cluster.Spec.MainContainer.ImageConfigs, versionString, processSettings.GetAllowTagOverride())
```

Comment:
Should we deprecate `processSettings.GetAllowTagOverride()` with this change?

Reply:
I think we can talk about that in the context of deprecating custom images in the pod specs in general.